packages/api-request-builder/src/delete.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+/**
+ * Set the `version` number to the internal state of the service instance
+ * in order to generate a delete uri
+ *
+ * @param  {number} version - The version of the item
+ * @throws if `version` is missing or not a number.
+ * @return {Object} The instance of the service, can be chained.
+ */
+
+export default function toDelete (version: number): Object {
+  if (!version || typeof version !== 'number')
+    throw new Error('Required version to delete missing or not valid')
```

Comment:
This error message looks strange to me: `Required version to delete missing or not valid`
What about to use something like this: `A resource version is invalid or missing`

But let's wait for others what do they think about it.